src/framework/components/gsplat/component.js

```diff
@@ -118,6 +123,12 @@ class GSplatComponent extends Component {
         }
     }
 
+    /**
+     * Gets the {@link GSplatInstance} on the component.
+     *
+     * @type {import('../../../scene/gsplat/gsplat-instance.js').GSplatInstance|null}
```

Comment:
and here

Reply:
Hmm, multiple instances. Perhaps a different PR for those.

Reply:
we have lots of instances of non-spaces around `|` in the engine, I'm not sure we have a strict rule around that at the moment.

Reply:
Yeah, we can standardize something in another PR.